cuda_bindings/docs/source/support.rst

```diff
@@ -23,9 +25,20 @@ The ``cuda.bindings`` module has the following support policy:
    module could require Cython layer users to rebuild their projects and update their pinning to
    this module.
 
+Free-threading Build Support
+----------------------------
+As of cuda-bindings 13.0.2/12.9.3, wheels for the `free-threaded interpreter`_ are shipped to PyPI.
```

Comment:
@oleksandr-pavlyk or @kkraus14 can correct me but I think it should be fairly easy to also ship on conda-forge. Maybe we don't have to emphasize PyPI.